kalite/playlist/urls.py

```diff
@@ -1,7 +1,14 @@
 from django.conf.urls import patterns, url
+from django.conf import settings
 
 
-urlpatterns = patterns(__package__ + '.views',
-                       url(r'^assign/$', 'assign_playlists', {}, 'assign_playlists'),
-                       url(r'^view/(?P<playlist_id>\w+)/$', 'view_playlist', {}, 'view_playlist'),
-)
+if "Nalanda" in settings.CONFIG_PACKAGE:
+    urlpatterns = patterns(__package__ + '.views',
+                           url(r'^assign/$', 'assign_playlists', {}, 'assign_playlists'),
+                           url(r'^view/(?P<playlist_id>\w+)/$', 'view_playlist', {}, 'view_playlist'),
+    )
+else:
```

Comment:
maybe we don't need these urls at all? Maybe a better pattern is to just leave `patterns` empty.